src/app/(proper_react)/MobileShell.tsx

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { ReactNode, useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+import { Session } from "next-auth";
+import styles from "./MobileShell.module.scss";
+import monitorLogo from "./images/monitor-logo.webp";
+import { CloseBigIcon, ListIcon } from "../components/server/Icons";
+import { useL10n } from "../hooks/l10n";
+import { PageLink } from "./PageLink";
+
+export type Props = {
+  children: ReactNode;
+  session: Session | null;
+};
+
+export const MobileShell = (props: Props) => {
+  const l10n = useL10n();
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <div
+      className={`${styles.wrapper} ${
+        isExpanded ? styles.hasOpenMenu : styles.hasClosedMenu
```

Comment:
It's indeed unused at this time, but I used it a bit in some initial experiments, and figured it's not unlikely we'll use it in the future, so I didn't remove it.

For the consistency I'm not sure, since obviously I'm not a native speaker, but I was under the impression that they're still in the same category: "the menu is open" vs. "the menu is closed"? i.e. adjective instead of past tense? But again, not a native speaker :sweat_smile: